index.src.html

```diff
@@ -459,6 +465,56 @@
     to guide the discovery process (by choosing a federated identity provider, BTLE device, etc).
   </div>
 
+  ### Mediation Requirements ### {#mediation-requirements}
+
+  When making a request via {{get(options)}}, developers can set a case-by-case requirement for
+  [=user mediation=] by choosing the appropriate {{CredentialMediationRequirement}} enum value.
+
+  Note: The [[#user-mediation]] section gives more detail on the concept in general, and its
+  implications on how the user agent deals with individual requests for a given origin).
+
+  <pre class="idl">
+    enum CredentialMediationRequirement {
+      "silent",
+      "optional",
+      "required"
+    };
+  </pre>
+  <dl dfn-for="CredentialMediationRequirement" dfn-type="enum-value">
+    :   <dfn>silent</dfn>
+    ::  User mediation is suppressed for the given operation. If the operation can be performed
+        without user involvement, wonderful. If user involvement is necessary, then the operation
+        will fail rather than involving the user.
+
+        Note: The intended usage is to support "Keep me signed-into this site" scenarios, where
+        a developer may wish to silently obtain credentials if a user should be automatically
+        signed in, but to delay bothering the user with a sign-in prompt until they actively
+        choose to sign-in.
+
+    :   <dfn>optional</dfn>
+    ::  If credentials can be handed over for a given operation without user mediation, they will
+        be. If [=requires user mediation|user mediation is required=], then the user agent will
+        involve the user in the decision.
+
+        Note: This is the default behavior for {{get()}}, and is intended to serve a case where a
+        developer has reasonable confidence that a user expects to start a sign-in operation. If
+        a user has just clicked "sign-in" for example, then they won't be surprised or confused to
+        see a [=credential chooser=].
+
+    :   <dfn>required</dfn>
+    ::  The user agent will not hand over credentials without [=user mediation=], even if the
+        [=requires user mediation=] flag is unset for an origin.
+
+        Note: For some operations, developers may wish to ensure that the user is involved, even if
+        they've otherwise chosen to stay signed-in. For example, a website might support sign-in
+        with multiple accounts at once, and it would be unfortunate if {{get()}} automatically
+        returned credentials without asking for disambiguation. In these cases, the developer can
```

Comment:
As discussed, I think it would be nice to make this scenario clearer.

Another point is that some password managers / authenticators may require a master password / biometric reauth as part of the mediation, so setting required to true would make a security difference.